src/p2p/JoinTypes.ts

```diff
@@ -42,11 +42,22 @@ export interface StandbyRefreshRequest {
   sign?: Types.Signature //sig of standby node 
 }
 
+export interface UnjoinRequest {
+  publicKey: string //pub key of the standby node
+}
+
+export type SignedUnjoinRequest = UnjoinRequest & Types.SignedObject
+
+export interface lostAfterSelectionRequest {
```

Comment:
lostAfterSelection is a value that we calculate, not one that we receive from gossip. this is similar to lostSyncing, where we dont have a request for it either. Rather, we just calculate if there are any lostSyncing or lostAfterSelection nodes based upon if a node has been syncing or selected for too long, respectively. so this isnt really needed. its better to follow the way lostSyncing does it just to keep conformity on how we deal with different types of data imo